src/lib/annotations/drawing/DrawingPath.js

```diff
@@ -0,0 +1,104 @@
+class DrawingPath {
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+    /** @property {Array} - The array of coordinates that form the path */
+    path = [];
+
+    /** @property {number} - The maximum X position of all coordinates */
+    maxX = -Infinity;
+
+    /** @property {number} - The maximum Y position of all coordinates */
+    maxY = -Infinity;
+
+    /** @property {number} - The minimum X position of all coordinates */
+    minX = Infinity;
+
+    /** @property {number} - The minimum Y position of all coordinates */
+    minY = Infinity;
+
+    /**
+     * Add position to coordinates and update the bounding box
+     *
+     * @param {number} xPos - xPosition to be part of the drawing
+     * @param {number} yPos - yPosition to be part of the drawing
+     * @return {void}
+     */
+    addCoordinate(xPos, yPos) {
+        if (!xPos || !yPos) {
+            return;
+        }
+        const x = parseFloat(xPos.toFixed(2));
+        const y = parseFloat(yPos.toFixed(2));
+        /* OPTIMIZE: We convert a number to a string using toFixed and then back to
+         *           a number. We might want to truncate only on annotation save.
+         */
+
+        if (x < this.minX) {
+            this.minX = x;
+        }
+
+        if (y < this.minY) {
+            this.minY = y;
+        }
+
+        if (y > this.maxY) {
+            this.maxY = y;
+        }
+
+        if (x > this.maxX) {
+            this.maxX = x;
+        }
```

Comment:
I feel that this is a little unclear (nested ternaries are not fun to read), and is taking care of the min and max in one statement when we will need to possibly assign 4 variables anyway. Doing a quick if-check for each of the variables doesn't seem too bad even though it looks like repeated logic.